adhocracy4/projects/migrations/0050_replace_point_field.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.17 on 2025-02-19 15:46
+
+import django.contrib.gis.db.models.fields
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("a4projects", "0049_project_geos_point"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="project",
+            name="point",
+        ),
+        migrations.AddField(
+            model_name="project",
+            name="point",
+            field=django.contrib.gis.db.models.fields.PointField(
+                blank=True,
+                help_text="Locate your project. Click inside the marked area or type in an address to set the marker. A set marker can be dragged when pressed.",
+                null=True,
+                srid=4326,
+                verbose_name="Can your project be located on the map?",
+            ),
+        ),
+    ]
```

Comment:
please separate these in two migrations, one for removing the field, the other for adding it, as it can give inconsistent schema errors again.